ts/src/test/static/response/hyperliquid.json

```diff
@@ -86,152 +85,152 @@
                     1
                 ],
                 "httpResponse": [
```

Comment:
@sc0Vu next time plz avoid formatting the entire file because it makes it impossible to do a proper diff